components/developer/rust/Makefile

```diff
@@ -27,43 +27,49 @@
 # Copyright 2023, Friedrich Kink
 #
 
+# This build is a mixture of configure and python style
+# it starts with configure for the initial setup 
+# the compiler build itself is intensively using python tools
+# therefor this patially python settings
+
 BUILD_BITS=		64
-USE_OPENSSL11=		yes
+OPENSSL_DEFAULT=	3.1
```

Comment:
Please never set `OPENSSL_DEFAULT` in component Makefile (or any other `*_DEFAULT` macro).